make/fat_lib_macros.mk

```diff
@@ -20,8 +20,10 @@ FAT_LIB_MACOSX_SDK_DIR := $(shell bash $(J2OBJC_ROOT)/scripts/sysroot_path.sh)
 FAT_LIB_IPHONE_SDK_DIR := $(shell bash $(J2OBJC_ROOT)/scripts/sysroot_path.sh --iphoneos)
 FAT_LIB_SIMULATOR_SDK_DIR := $(shell bash $(J2OBJC_ROOT)/scripts/sysroot_path.sh --iphonesimulator)
 FAT_LIB_WATCH_SDK_DIR := $(shell bash $(J2OBJC_ROOT)/scripts/sysroot_path.sh --watchos)
+FAT_LIB_TV_SDK_DIR := $(shell bash $(J2OBJC_ROOT)/scripts/sysroot_path.sh --appletvos)
+FAT_LIB_TVSIMULATOR_SDK_DIR := $(shell bash $(J2OBJC_ROOT)/scripts/sysroot_path.sh --appletvsimulator)
 
-FAT_LIB_MACOSX_FLAGS = $(FAT_LIB_OSX_FLAGS) -DJ2OBJC_BUILD_ARCH=x86_64 \
+FAT_LIB_MACOSX_FLAGS = $(FAT_LIB_OSX_FLAGS) -DJ2OBJC_BUILD_ARCH=x86_64 -mmacosx-version-min=10.7 \
```

Comment:
Building an OS X application that makes use of J2ObjC libs and legitimately targets a previous version of OS X, produces lots of warnings such as:

```
ld: warning: object file (/Users/gbertani/[REDACTED]/j2objc/lib/macosx/libjre_emul.a(BufferedOutputStream.o)) was built for newer OSX version (10.11) than being linked (10.9)
ld: warning: object file (/Users/gbertani/[REDACTED]/j2objc/lib/macosx/libjre_emul.a(BufferedReader.o)) was built for newer OSX version (10.11) than being linked (10.9)
ld: warning: object file (/Users/gbertani/[REDACTED]/j2objc/lib/macosx/libjre_emul.a(BufferedWriter.o)) was built for newer OSX version (10.11) than being linked (10.9)
```

There's no reason to build J2ObjC targeting the latest OS X version if code is compatible with OS X APIs down to 10.7. This flag just lowers the target to an API level comparable to that targeted for iOS.